helm/cas-ggircs/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: cas-ggircs
+description: Helm chart for the GGIRCS database and ETL
+type: application
+version: 0.1.0
+appVersion: 0.1.0
+dependencies:
+- name: cas-postgres
+  version: "0.2.0"
+  repository: https://bcgov.github.io/cas-postgres/
```

Comment:
woot woot! the `cas-postgres` chart will need to have all its dependencies published (no file dependencies) for this to work

Reply:
see https://github.com/bcgov/cas-postgres/pull/50/files#r436111322

Reply:
It's been published to the ghpages, and I'm pretty sure this works as-is, with the file dependency

Reply:
I'll have to look into why that works now; didn't used to